lib/Ohmbrewer_Screen.cpp

```diff
@@ -360,3 +369,87 @@ unsigned long Ohmbrewer::Screen::displayStatusUpdate(char *statusUpdate) {
 
     return micros() - start;
 }
+
+/**
+ * Checks for a touch event and triggers actions 
+ *  if the the touch was on a screen "button".
+ * @returns Time it took to run the function
+ */
+unsigned long Ohmbrewer::Screen::captureButtonPress() {
+    unsigned long start = micros();
+    char printx [10];
+	char printy [10];
+	char status [40];
+	
+	// a point object holds x y and z coordinates
+  	TSPoint p = ts->getPoint();
+  	//According to Particle forums, the read below is necessary to get touch to work on Photon
+  	ts->readTouchY();
```

Comment:
Did you figure out if this read has to be done every single time you want a new point or if it can be done once after boot up? 'Cause if it's the latter, we should be able to move this into the constructor.